index.js

```diff
@@ -326,14 +335,14 @@ bluebird.all([
         var attribution = _.filter(licenseInfos, licenseInfo => {
             return !licenseInfo.ignore;
         }).map(licenseInfo => {
-            return [
-                licenseInfo.name,
-                `${licenseInfo.version} <${licenseInfo.url}>`,
-                licenseInfo.licenseText || `license: ${licenseInfo.license}${os.EOL}authors: ${licenseInfo.authors}`
-            ].join(os.EOL);
+            if(licenseInfo.name != undefined) {
+                return [licenseInfo.name,`${licenseInfo.version} <${licenseInfo.url}>`,
+                licenseInfo.licenseText || `license: ${licenseInfo.license}${os.EOL}authors: ${licenseInfo.authors}`].join(os.EOL);
+            }
         }).join(`${os.EOL}${os.EOL}******************************${os.EOL}${os.EOL}`);
```

Comment:
because you're not returning an entry if the check on line 338 is falsy, the entry in the returned array would be `undefined`. Please add a comment describing this where the `else` block would be, and add a filter before joining on line 342.